### ISSUE TITLE
Proposal for texture streamlining in next release (8.5)

### ISSUE THREAD

**Issue by ja-to** [WolvenKit/WolvenKit#631]
The latest WolvenKit release has a good texture workflow, however the application isn't set up to take advantage out of the box. There's a few changes that could be made to make texture modding more intuitive.

- [x]  **Set "Use existing file" to false by default**
Currently default textures are set to rebuild using an existing file which has a few drawbacks.
    - When importing DDS the compression method must match or the texture will be deep fried. (most common issue)
    - When importing TGA the Import tool freezes and fails to generate a valid XBM for most textures.
    -  When changing texture resolution, the XBM must be manually updated.
    
- [x]  **Set default export format to TGA** 
Currently the default image export is DDS which is lossy and handled very poorly by most image applications.

- [x]  **Expose isGamma boolean for imports**
When XBM's are generated from scratch, the **isGamma** boolean is set to false by default. Color textures (diffuse, billboards, etc.) must have this boolean set **true** to display properly as sRGB. It would be a significant time saver compared to opening the XBM post-import to set the boolean.

- [x] #937
The **isGamma** boolean could be automatically set by a huge lookup table (Legacy WolvenKit) or use the texture naming scheme (...d01 is true, ...n01 is false).

**Comment by ja-to** [comment]
The flow chart below illustrates the need for changes to the default texture options. Each red-tinted node represents an opportunity for failure within the DDS pipeline. The proposed TGA pipeline is significantly more robust, and less error prone in comparison.

![TGA vs DDS texture workflow](https://user-images.githubusercontent.com/65016231/148646451-16ff173a-1224-4bb9-93dd-0aabad10c677.png)

Updates on the four issues:

1. Use Existing File/Keep can easily be set to false by default, however the current implementation is a global setting that affects all file types. Ideally textures should use a unique Keep setting.
See [ImportExportArguments.cs](https://github.com/WolvenKit/WolvenKit/blob/master/WolvenKit.Common/Model/Arguments/ImportExportArguments.cs)

2. I implemented this already by reordering [Enums.cs](https://github.com/WolvenKit/WolvenKit/blob/master/WolvenKit.Core/Enums.cs) image extensions.

3. Support for isGamma has been implemented already as a boolean.
See [commit #633](https://github.com/WolvenKit/WolvenKit/commit/397214cb5a17095f47ff62d44f7f712dc97382ca)

4. Currently isGamma is implemented as a boolean within the app, however ideally the app should use a drop down with three options:
Automatic (default)
True
False
